### PR TITLE
Issue 1432 - Added quest 7670 to solve issue

### PR DESCRIPTION
**Pull request by NguyenMartin**
Quest [Lord Grayson Shadowbreaker] (7670) was missing from the database. Solved issue #1432 by adding the quest in.

### CODE REVIEW

File: Database/Corrections/QuestieQuestFixes.lua [hunk 1]
```diff
@@ -566,5 +567,18 @@ function QuestieQuestFixes:Load()
             [QuestieDB.questKeys.exclusiveTo] = {5887,5888,5889,5890,},
             [QuestieDB.questKeys.specialFlags] = 0,
         },
+        [7670] = { -- #1432
+        [QuestieDB.questKeys.name] = "Lord Grayson Shadowbreaker",
+        [QuestieDB.questKeys.startedBy] = {{5149},nil,nil,},
+        [QuestieDB.questKeys.finishedBy] = {{928},nil,},
+        [QuestieDB.questKeys.requiredLevel] = 60,
+        [QuestieDB.questKeys.questLevel] = 60,
+        [QuestieDB.questKeys.requiredRaces] = 5,
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]: On [Wowhead](https://classic.wowhead.com/quest=7670/lord-grayson-shadowbreaker) it is marked as an alliance quest an therefore it should be:

```suggestion
        [QuestieDB.questKeys.requiredRaces] = 77,
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 2]
```diff
@@ -566,5 +567,18 @@ function QuestieQuestFixes:Load()
             [QuestieDB.questKeys.exclusiveTo] = {5887,5888,5889,5890,},
             [QuestieDB.questKeys.specialFlags] = 0,
         },
+        [7670] = { -- #1432
+        [QuestieDB.questKeys.name] = "Lord Grayson Shadowbreaker",
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
            [QuestieDB.questKeys.name] = "Lord Grayson Shadowbreaker",
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 3]
```diff
@@ -566,5 +567,18 @@ function QuestieQuestFixes:Load()
...
+        [QuestieDB.questKeys.startedBy] = {{5149},nil,nil,},
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
            [QuestieDB.questKeys.startedBy] = {{5149},nil,nil,},
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 4]
```diff
@@ -566,5 +567,18 @@ function QuestieQuestFixes:Load()
...
+        [QuestieDB.questKeys.finishedBy] = {{928},nil,},
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
            [QuestieDB.questKeys.finishedBy] = {{928},nil,},
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 5]
```diff
@@ -566,5 +567,18 @@ function QuestieQuestFixes:Load()
...
+        [QuestieDB.questKeys.requiredLevel] = 60,
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
            [QuestieDB.questKeys.requiredLevel] = 60,
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 6]
```diff
@@ -566,5 +567,18 @@ function QuestieQuestFixes:Load()
...
+        [QuestieDB.questKeys.questLevel] = 60,
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
            [QuestieDB.questKeys.questLevel] = 60,
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 7]
```diff
@@ -566,5 +567,18 @@ function QuestieQuestFixes:Load()
...
+        [QuestieDB.questKeys.requiredRaces] = 5,
+        [QuestieDB.questKeys.requiredClasses] = 2,
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
            [QuestieDB.questKeys.requiredClasses] = 2,
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 8]
```diff
@@ -566,5 +567,18 @@ function QuestieQuestFixes:Load()
...
+        [QuestieDB.questKeys.objectivesText] = {"Speak with Lord Grayson Shadowbreaker in Stormwind's Cathedral District."},
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
            [QuestieDB.questKeys.objectivesText] = {"Speak with Lord Grayson Shadowbreaker in Stormwind's Cathedral District."},
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 9]
```diff
@@ -566,5 +567,18 @@ function QuestieQuestFixes:Load()
...
+        [QuestieDB.questKeys.nextQuestInChain] = 7637,
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
            [QuestieDB.questKeys.nextQuestInChain] = 7637,
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 10]
```diff
@@ -566,5 +567,18 @@ function QuestieQuestFixes:Load()
...
+        [QuestieDB.questKeys.exclusiveTo] = {7638,},
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
            [QuestieDB.questKeys.exclusiveTo] = {7638,},
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 11]
```diff
@@ -566,5 +567,18 @@ function QuestieQuestFixes:Load()
...
+        [QuestieDB.questKeys.zoneOrSort] = -141
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
            [QuestieDB.questKeys.zoneOrSort] = -141
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 12]
```diff
@@ -566,5 +567,18 @@ function QuestieQuestFixes:Load()
...
+    },
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
        },
```

**Comment by BreakBB** [comment]
@NguyenMartin Thanks for another PR. You're really helping out atm! 👍 I added some suggestions, you should commit. Everything else looks fine for me :)

**Comment by dimsno** [comment]
Thanks for looking over the correction! I've committed all of your recommendations.

**Comment by NguyenMartin** [comment]
Thanks for your feedback! I've gone ahead and pushed the suggested corrections!

**Pull request merged**